### ISSUE TITLE
Responsive design glitch

### ISSUE THREAD

**Issue by RobinIsTheBird** [OpenTreeMap/otm-core#2737]
See the gray stripe beneath the Add a Tree, Add a Mitigator bar. Happens between 768 and 990 pixel widths.

![responsiveglitch](https://cloud.githubusercontent.com/assets/1473263/18012789/07516916-6b89-11e6-955c-866a902f325d.png)

**Comment by RobinIsTheBird** [comment]
Also happens at 600px with the `embed` query param.

**Comment by RobinIsTheBird** [comment]
Related - not sure if it belongs in a separate issue, is a problem at 800px depending on # of trees and terminology customization. Now that the planting site count is called "empty planting sites", the problem is even easier to provoke, even with the `embed` query param, which hides the `Export Search Results` button.

![otm 800w statusline layout issue](https://cloud.githubusercontent.com/assets/1473263/18601578/a24ed99a-7c31-11e6-902b-22bdb3fdf251.png)
